.drone.yml

```diff
@@ -28,31 +28,33 @@ steps:
     event:
     - tag
 
-- name: push
-  image: rancher/hardened-build-base:v1.15.8b5
+- name: manifest
```

Comment:
lets just comment manifest out until we get multi-arch going and get firm grasp on handling situation.